docs/source/en/training/ip_adapter.md

```diff
@@ -0,0 +1,121 @@
+# IP Adapter Training Example 
```

Comment:
> I think this can be moved to the README.md as well if there's any difference in the content. This is because for projects within `research_projects` folder, we typically don't include a doc

The files are same, so I removed the file.